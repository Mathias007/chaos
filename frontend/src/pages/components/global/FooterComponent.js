import React from "react";

import styles from "../../../styles/styles";

import { Layout } from "antd";
const { Footer } = Layout;

export default function FooterComponent(props) {
    const generateActualYear = () => {
        return new Date().getFullYear();
    };

    return (
        <Footer style={styles.footer}>
            <p>
                Â© 2019-{generateActualYear()}{" "}
                <a href="https://egildia.pl">egildia.pl</a>
                <br />
                Created by Mateusz Mathias Stawowski
            </p>
        </Footer>
    );
}
