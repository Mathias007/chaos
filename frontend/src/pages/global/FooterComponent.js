import React, { Component } from "react";
import { Layout } from "antd";

const { Footer } = Layout;

class FooterComponent extends Component {
    render() {
        return (
            <Footer style={{ textAlign: "center" }}>
                egildia.pl Â© 2019-2020 Mateusz Mathias Stawowski
            </Footer>
        );
    }
}

export default FooterComponent;
