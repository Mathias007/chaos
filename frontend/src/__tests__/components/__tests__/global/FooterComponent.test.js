// Auto-generated do not edit

/* eslint-disable import/no-extraneous-dependencies */
/* eslint-disable no-undef */
import React from "react";
import renderer from "react-test-renderer";
import FooterComponent from "../../FooterComponent";

describe("FooterComponent test", () => {
    it("FooterComponent should match snapshot", () => {
        const component = renderer.create(<FooterComponent />);
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
