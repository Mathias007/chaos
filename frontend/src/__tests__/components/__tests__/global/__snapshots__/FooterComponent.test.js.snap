// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterComponent test FooterComponent should match snapshot 1`] = `
<footer
  className="ant-layout-footer"
  style={
    Object {
      "textAlign": "center",
    }
  }
>
  <p>
    Â© 2019-
    2020
     
    <a
      href="https://egildia.pl"
    >
      egildia.pl
    </a>
    <br />
    Created by Mateusz Mathias Stawowski
  </p>
</footer>
`;
